eplaunch/interface/dialog_external_viewers.py

```diff
@@ -0,0 +1,128 @@
+from platform import system
```

Comment:
All dialog classes named `dialog_*`.  All of these use Tk and follow the same basic patterns.